Ruby package build failures

There are two errors in the Ruby package build not covered in the README; I'm not sure if I'm doing something wrong or just don't have the prereqs installed on my host.

First one (from `make`):

```
installing default gems:      /var/lib/gems/2.2.0 (build_info, cache, doc, extensions, gems, specifications)
/home/pooka/rumpkernel/rumprun-packages/ruby/build/lib/fileutils.rb:249:in `mkdir': Permission denied @ dir_s_mkdir - /var/lib/gems (Errno::EACCES)
	from /home/pooka/rumpkernel/rumprun-packages/ruby/build/lib/fileutils.rb:249:in `fu_mkdir'
	from /home/pooka/rumpkernel/rumprun-packages/ruby/build/lib/fileutils.rb:226:in `block (2 levels) in mkdir_p'
	from /home/pooka/rumpkernel/rumprun-packages/ruby/build/lib/fileutils.rb:224:in `reverse_each'
	from /home/pooka/rumpkernel/rumprun-packages/ruby/build/lib/fileutils.rb:224:in `block in mkdir_p'
	from /home/pooka/rumpkernel/rumprun-packages/ruby/build/lib/fileutils.rb:210:in `each'
	from /home/pooka/rumpkernel/rumprun-packages/ruby/build/lib/fileutils.rb:210:in `mkdir_p'
	from ./tool/rbinstall.rb:186:in `makedirs'
	from ./tool/rbinstall.rb:293:in `prepare'
	from ./tool/rbinstall.rb:661:in `block in <main>'
	from ./tool/rbinstall.rb:757:in `call'
	from ./tool/rbinstall.rb:757:in `block in <main>'
	from ./tool/rbinstall.rb:754:in `each'
	from ./tool/rbinstall.rb:754:in `<main>'
uncommon.mk:253: recipe for target 'do-install-nodoc' failed
make[1]: *** [do-install-nodoc] Error 1
make[1]: Leaving directory '/home/pooka/rumpkernel/rumprun-packages/ruby/build'
Makefile:18: recipe for target 'build/ruby' failed
make: *** [build/ruby] Error 2
```

Second one (from `make images/examples.iso`):

```
cd examples && bundle install --path=gems
/bin/sh: 1: bundle: not found
Makefile:33: recipe for target 'images/examples.iso' failed
make: *** [images/examples.iso] Error 127
```

--- a/ruby/Makefile
+++ b/ruby/Makefile
@@ -13,11 +13,10 @@ bin/ruby: build/ruby
 	mkdir -p bin
 	rumprun-bake hw_generic bin/ruby.bin build/rubydist/bin/ruby
 
-# --with-out-ext=fiddle,bigdecimal,continuation,coverage,date,dbm,-test*;
 build/ruby: build/Makefile
 	(cd build; \
-	./configure --enable-static --disable-shared --host=$(RUMPRUN_TOOLCHAIN_TUPLE) --with-static-linked-ext --prefix=$(shell pwd)/build/rubydist --disable-install-doc --with-out-ext=-test* LDFLAGS="-Bstatic -static"; \
-	$(MAKE) --debug; make install)
+	./configure --enable-static --disable-shared --host=$(RUMPRUN_TOOLCHAIN_TUPLE) --with-static-linked-ext --prefix=/usr --disable-install-doc --with-out-ext=-test* LDFLAGS="-Bstatic -static"; \
+	$(MAKE) --debug; make install DESTDIR=rubydist)
 
 build/Makefile:
 	mkdir -p build

